bananas_cli/session.py

```diff
@@ -27,6 +27,8 @@ async def stop(self):
 
     async def _read_response(self, response):
         if response.status in (200, 201, 400, 404):
+            if response.content_type == "text/html":
+                return 302, response.url
```

Comment:
I need a bit more detail what this does and why it is needed. For sure, if it is correct, it requires a comment in code explaining why this is there. But it feels off .. like something else is wrong :)

Reply:
With github mode, bananas-api will redirect the user to the login page: 
https://github.com/OpenTTD/bananas-api/blob/7e9f4cf8c79cd6954607c8e43595d3c4d27e516e/bananas_api/user/github.py#L48

However, in developer mode, bananas-api will just return a login form: 
https://github.com/OpenTTD/bananas-api/blob/7e9f4cf8c79cd6954607c8e43595d3c4d27e516e/bananas_api/user/developer.py#L16

The CLI tool expects to be redirected in case the request was succesful, so this is what is being simulated here.

The _proper_ solution would be to also redirect in developer mode. Once I get more comfortable writing Python and with the bananas-api project I may be able to fix that, but not right now. I could create an issue in bananas-api and refer to that in a comment here perhaps?